console/src/components/WelcomePopup/WelcomePopup.module.less

```diff
@@ -89,9 +93,9 @@
 		.contentCell {
 			margin: 12px 84px;
 
-			&:last-child {
-				margin-bottom: 48px;
-			}
+			//&:last-child {
+			//	margin-bottom: 48px;
+			//}
```

Comment:
Do we need to keep these commented codes? If it's not needed, please remove them.